src/Core/Jobs/BaseJobsHostedService.cs

```diff
@@ -18,25 +18,45 @@ public abstract class BaseJobsHostedService : IHostedService, IDisposable
         protected readonly ILogger _logger;
 
         private IScheduler _scheduler;
+        protected GlobalSettings _globalSettings;
 
         public BaseJobsHostedService(
+            GlobalSettings globalSettings,
             IServiceProvider serviceProvider,
             ILogger logger,
             ILogger<JobListener> listenerLogger)
         {
             _serviceProvider = serviceProvider;
             _logger = logger;
             _listenerLogger = listenerLogger;
+            _globalSettings = globalSettings;
         }
 
         public IEnumerable<Tuple<Type, ITrigger>> Jobs { get; protected set; }
 
         public virtual async Task StartAsync(CancellationToken cancellationToken)
         {
-            var factory = new StdSchedulerFactory(new NameValueCollection
+            var props = new NameValueCollection
             {
-                { "quartz.serializer.type", "binary" }
-            });
+                {"quartz.serializer.type", "binary"},
+            };
+
+            if (!_globalSettings.SelfHosted)
```

Comment:
2 checks should go here, please add a check to ensure `_globalSettings.SqlServer.JobSchedulerConnectionString` is actually set and not empty/null. That way we can merge and deploy this whenever w/o risk before the other database is stood up/operationalized and connection string info added in production.